docs/sources/installation.md

```diff
@@ -4,7 +4,53 @@ Here are some requirements and guides to contributing to the Girl Develop It web
 
 ---
 
-### Required installations
+### Vagrant installation instructions
+
+Using Vagrant is the suggested way to do development for the website. It provides a standard environment that works across platforms.
+
+#### 1. Install VirtualBox
+VirtualBox is a cross-platform virtualization application (see: [the VirtualBox manual](https://www.virtualbox.org/manual/ch01.html)). That means it can install multiple virtual machines at the same time, which is great for development. You can download the latest version on [the VirtualBox website](https://www.virtualbox.org/wiki/Downloads).
+
+#### 2. Install Vagrant
+Vagrant is a wonderful tool to help you build complete development environments. Download the latest version on [the Vagrant website](https://www.vagrantup.com/downloads.html).
+
+#### 3. Clone the gdi-website Git repository
+Clone the Git repository into your local development folder:
+
+```sh
+git clone git@github.com:girldevelopit/gdi-website.git
+cd gdi-website
+```
+
+#### 4. Start Vagrant
+Depending on your machine, this could take a while. Once it's started, you'll ssh into the Vagrant box and run later commands from that environment.
+
+```sh
+vagrant up
+vagrant ssh
+```
+
+#### 5. Install and initialize the development environment
+Run the following:
+
+```sh
+cd /opt/gdi/development
+bundle install
+rake db:create db:migrate db:seed
+rails server --binding=0.0.0.0
```

Comment:
I found that this was necessary to connect. Found the solution here: https://stackoverflow.com/questions/27627286/cant-connect-localhost3000-ruby-on-rails-in-vagrant

Reply:
Agree!